documentation/src/docs/asciidoc/release-notes-5.0.0-M5.adoc

```diff
@@ -54,4 +54,5 @@ on GitHub.
 
 ===== Bug Fixes
 
-* ❓
+* The `IsPotentialJUnit4TestClass` predicate incorrectly filtered out static nested classes.
+  It now let's them through properly
```

Comment:
I think that's a bit too technical. We should try to write this from the point of view of a Vintage user. It should be "static member classes" because that excludes local and anonymous classes. Moreover, the last sentence is missing a full stop/period.